### PR TITLE
update Compose release notes page for v2.20.1 and v2.20.2

### PR DESCRIPTION
**Pull request by glours**
### Proposed changes
Update Compose release notes page with version `v2.20.1` and `v2.20.2`

### CODE REVIEW

File: compose/release-notes.md [hunk 1]
```diff
@@ -10,6 +10,23 @@ redirect_from:
 
 For more detailed information, see the [release notes in the Compose repo](https://github.com/docker/compose/releases/).
 
+## 2.20.2
+{% include release-date.html date="2023-07-19" %}
+
+### Bug fixes and enhancements
+- Add support of `depends_on.required` attribute
+- Fixed an issue about build trying to push unnamed service images
+- Fixed a bug with target secret path on Windows
```
Comment (milas) [inline compose/release-notes.md:19]:
```suggestion
- Fixed a bug with target secret path on Windows
- Fixed a bug resolving build context path for services using `extends.file`
```

That fix comes from `compose-go` but was reported as docker/for-mac#6912, so I think might be worth an explicit callout.

**Review by milas — APPROVED** [review]
LGTM! Thanks for including v2.20.1

File: compose/release-notes.md [hunk 2]
```diff
@@ -10,6 +10,23 @@ redirect_from:
 
 For more detailed information, see the [release notes in the Compose repo](https://github.com/docker/compose/releases/).
 
+## 2.20.2
+{% include release-date.html date="2023-07-19" %}
+
+### Bug fixes and enhancements
+- Add support of `depends_on.required` attribute
```
Comment (aevesdocker) [inline compose/release-notes.md:17]:
```suggestion
- Added support for the `depends_on.required` attribute.
```

File: compose/release-notes.md [hunk 3]
```diff
@@ -10,6 +10,23 @@ redirect_from:
...
+- Fixed an issue about build trying to push unnamed service images
```
Comment (aevesdocker) [inline compose/release-notes.md:18]:
```suggestion
- Fixed an issue where build tries to push unnamed service images.
```

File: compose/release-notes.md (hunk 1 above)
Comment (aevesdocker) [inline compose/release-notes.md:19]:
```suggestion
- Fixed a bug meant the target secret path on Windows was not checked.
```

File: compose/release-notes.md [hunk 4]
```diff
@@ -10,6 +10,24 @@ redirect_from:
 
 For more detailed information, see the [release notes in the Compose repo](https://github.com/docker/compose/releases/).
 
+## 2.20.2
+{% include release-date.html date="2023-07-19" %}
+
+### Bug fixes and enhancements
+- Added support for the `depends_on.required` attribute.
+- Fixed an issue where build tries to push unnamed service images.
+- Fixed a bug with target secret path on Windows
```
Comment (aevesdocker) [inline compose/release-notes.md:19]:
```suggestion
- Fixed a bug which meant the target secret path on Windows was not checked.
```

File: compose/release-notes.md [hunk 5]
```diff
@@ -10,6 +10,24 @@ redirect_from:
...
+- Fixed a bug resolving build context path for services using `extends.file`
```
Comment (aevesdocker) [inline compose/release-notes.md:20]:
```suggestion
- Fixed a bug resolving build context path for services using `extends.file`.
```

**Review by aevesdocker — APPROVED** [review]

**Pull request merged**